Libraries/WebRTC/nvencoder.patch

```diff
@@ -7636,7 +7636,7 @@ index 0000000..418548a
 +    memset(&m_lockBitstreamData, 0, sizeof(m_lockBitstreamData));
 +    SET_VER(m_lockBitstreamData, NV_ENC_LOCK_BITSTREAM);
 +	m_lockBitstreamData.outputBitstream = pEncodeBuffer->stOutputBfr.hBitstreamBuffer;
-+	m_lockBitstreamData.doNotWait = false;
++	m_lockBitstreamData.doNotWait = true;

```

Comment:
what's the impact of this change? I am guessing it changes the behavior of some blocking read operation?

Reply:
Yes, this change makes the encoding task operates asynchronously instead of blocking. As a result, the capturing stream becomes much more stable.